CHANGELOG.rst

```diff
@@ -2,6 +2,43 @@
 Release Notes
 =============
 
+.. _Release Notes_7.25.1:
+
+7.25.1
+======
+
+.. _Release Notes_7.25.1_Prelude:
+
+Prelude
+-------
+
+Release on: 2021-01-26
```

Comment:
reminder to merge the changelog and update the release date when 7.25.1 is actually out